### PR TITLE
Namespace show all

### PR DESCRIPTION
**Pull request by amolenaar**
### PR Type
What kind of change does this PR introduce?

- [X] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes

### What is the current behavior?

Issue Number: #334 

### What is the new behavior?

All namespaced information is shown.

### Does this PR introduce a breaking change?
- [ ] Yes
- [X] No

### CODE REVIEW

**Comment by amolenaar** [comment]
The models shows all elements, which is something that needs fixing:

image

**Comment by danyeaw** [comment]
I really like this approach. I think autoformatting the name using the umlfmt module and removing the icon were nice touches.

A few things I have been thinking about:
1. Should we have tools in the tool palette under the blocks section to create a Composition Relationship and Aggregation Relationship directly as a shortcut to creating an association, adding navigation, and setting shared or composite aggregation?
2. Should we automatically name the Property of these types of relationships? For example, I know some of the other tools name the property name the same as the class name, except as lower camel case instead of upper camel case like the class name.

Right now if you create an unnamed association it just shows up as a blank row in the namespace:
image

**Comment by amolenaar** [comment]
> 1. Should we have tools in the tool palette under the blocks section to create a Composition Relationship and Aggregation Relationship directly as a shortcut to creating an association, adding navigation, and setting shared or composite aggregation?

This makes sense, I think, since now we need to head over to the editor to change the association ends.

> 2. Should we automatically name the Property of these types of relationships? For example, I know some of the other tools name the property name the same as the class name, except as lower camel case instead of upper camel case like the class name.

For composite and aggregate associations it makes sense, for sure. Since there already is this directional aspect to it, with navigability.

> Right now if you create an unnamed association it just shows up as a blank row in the namespace:

Hmm.. It should at least show something. Even `<None>` would be better than nothing.

**Comment by amolenaar** [comment]
We still need to deal with relationships, too.

**Comment by amolenaar** [comment]
I made attribute formatting a little more liberal, now it will render a string, even when there is no name for the property. Also it will parse associations with a `: Type` defined and just ignore that type (needed for editing in the tree view).

**Comment by amolenaar** [comment]
Let's handle relationships in a different PR.

File: gaphor/UML/tests/test_umllex.py [hunk 1]
```diff
@@ -145,6 +145,22 @@ def test_parse_association_end_derived_end(factory):
...
+    p = factory.create(UML.Property)
+    p.association = a
+
+    UML.parse(p, "end: TypeVal")
+    assert "end" == p.name
+    assert not p.typeValue
+    assert not p.lowerValue
+    assert not p.upperValue
+    assert not p.defaultValue
+    assert not p.typeValue
```
Comment (danyeaw) [inline gaphor/UML/tests/test_umllex.py:161]: Is there supposed to be two assert statements for `p.typeValue`?

Comment (amolenaar) [reply gaphor/UML/tests/test_umllex.py]: Fixed

**Pull request merged**